doc/aws.md

```diff
@@ -26,6 +26,23 @@ Note that if you do not use Route53 DNS you MUST provide one or more allowed COR
 
 In these templates the Load Balancer expects to provide access to the LRS via HTTPS/443. You will need to either acquire a free Amazon Certificate Manager cert (highly recommended) or import your own cert from another CA into ACM for use in the deployment.
 
+#### S3 Bucket (Non-US Regions Only)
+
+SQL LRS requires a Lambda function to create the application-level database user. For the following regions the code for this function is hosted by Yet Analytics and provided to the template automatically:
```

Comment:
Maybe make a note to leave the DBInitFnBucketOverride /Key param blank.

Reply:
Look further down in the step-by-step params, it does just that.